src/integrationtest/java/com/ericsson/ei/frontend/CommonSteps.java

```diff
@@ -89,8 +88,8 @@ public void aggregated_object_created() throws IOException, TimeoutException {
         String eventFilePath = Paths.get(RESOURCE_PATH, EIFFEL_EVENT_FILE).toString();
         String eventFileContent = FileUtils.readFileToString(new File(eventFilePath), "UTF-8");
 
-        AMQPCommunication amqp = new AMQPCommunication(rabbitHost, rabbitPort, rabbitUsername, rabbitPassword);
-        assertEquals(true, amqp.produceMessage(eventFileContent, rabbitExchange, rabbitKey));
+        AMQPCommunication amqp = new AMQPCommunication(rabbitmqHost, rabbitmqPort, rabbitmqUsername, rabbitmqPassword);
```

Comment:
remove everything related to rabbitmq

Reply:
This is in the integration test though. It sends an event to the message bus to make an aggregation happen.

Reply:
ok! I missed that.